packages/addon-dev/src/rollup.ts

```diff
@@ -16,20 +17,28 @@ export class Addon {
   constructor(params: { srcDir?: string; destDir?: string } = {}) {
     this.#srcDir = params.srcDir ?? 'src';
     this.#destDir = params.destDir ?? 'dist';
+
+    let pkg = readJsonSync('package.json');
+    delete pkg['ember-addon']['app-js'];
+    writeJsonSync('package.json', pkg, { spaces: 2 });
   }
 
   // Given a list of globs describing modules in your srcDir, this generates
   // corresponding appTree modules that contain reexports, and updates your
   // package.json metadata to list them all.
   appReexports(
     patterns: string[],
-    opts: { mapFilename?: (fileName: string) => string } = {}
+    opts: {
+      mapFilename?: (fileName: string) => string;
+      defaultExport?: string;
+    } = {}
   ): Plugin {
     return appReexports({
       from: this.#srcDir,
       to: this.#destDir,
       include: patterns,
       mapFilename: opts.mapFilename,
+      defaultExport: opts.defaultExport || '{ default }',
```

Comment:
The default value of this option should be in rollup-app-reexports.ts itself, not this callsite.